library/include/borealis/view.hpp

```diff
@@ -351,6 +358,29 @@ class View
         return nullptr;
     }
 
+    /**
+     * Return the child at xpos and ypos (in screen coordinated), or null if
+     * there is none at the position.
+     */
+    virtual View* getChildViewAtTouch(float xpos, float ypos);
+
+    /**
+     * Self explanatory.
+     */
+    virtual bool isFocusable() const
```

Comment:
I am not a fan of redefining `isFocusable` - we already have a definition of something focusable, it's "getDefaultFocus() returns this". I think this new method should be called "isTouchable" or "isFocusableByTouch" maybe?

Reply:
I like the "isFocusableByTouch()," I'll remember to do that when I get the chance.

Reply:
Actually no, scratch that. I think isTouchable describes its purpose better.

Reply:
Quick question:
Should every focusable be a touchable? If so then I think it'd be best to just scrap the isFocusable/isFocusableByTouch idea that I had and just check if getDefaultFocus is equivalent to the object that is being check.

Reply:
Every focusable is a touchable but not every touchable is focusable (bottom-right hints for instance are touchable and not focusable)

Reply:
Lists itself (not the items in it) are not focusable but touchable